packaging/nuget/NServiceBus.Metrics.PerformanceCounters.nuspec

```diff
@@ -19,6 +19,6 @@
     </dependencies>
   </metadata>
   <files>
-    <file src="..\..\binaries\NServiceBus.Metrics.PerformanceCounters.???" target="lib\net452" />
+    <file src="..\..\binaries\net452\NServiceBus.Metrics.PerformanceCounters.???" target="lib\net452" />
```

Comment:
Is `net452` folder work with https://github.com/Particular/NServiceBus.Metrics.PerformanceCounters/pull/31/files#diff-7f82810f15329f240ed8c7e937b383dbR15 ?

Reply:
I don't see the line you anchored.  Do you mean `<OutputPath>..\..\binaries\</OutputPath>` ? If this is the one, then yes, this will work as the new build system puts binaries into respective framework dirs. Does it answer your question?